app/views/submissions/show.html.haml

```diff
@@ -152,3 +152,24 @@
           %td= @submission.latest_modifier
           %td= @submission.latest_modification_time
           %td{class: "#{Submission::STATUS_MAPPING[@submission.status]}"}= @submission.status
+  %h3 Widomości związane ze zgłoszeniem:
+  %table.table.table-hover.table-admin
+    %thead
+      %tr
+        %th Nadawca
+        %th Odbiorca
+        %th Data
+        %th Tytuł
+        %th Treść
+    %tbody
+      -  # Jeśli maile sa przypisane do zgłoszeń
+      - if defined?(@submission.SubmissionMail.last)
```

Comment:
można to pominąć, poza tym powinno być `@submission.submission_mails.last` (albo po zmianie nazwy asocjacji `@submission.mails.last`)